Places/View Controllers/ItinerariesTableViewController.m

```diff
@@ -24,6 +25,11 @@ @implementation ItinerariesTableViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
+    // Initialize a UIRefreshControl
+    self.refreshControl = [[UIRefreshControl alloc] init];
+    [self.refreshControl addTarget:self action:@selector(fetchItineraries) forControlEvents:UIControlEventValueChanged];
+    [self.tableView insertSubview:self.refreshControl atIndex:0];
```

Comment:
I find it interesting that this works. I'm used to setting `tableViewHeader` to the refresh control. Maybe `UITableView`'s layout logic just finds the first subview of kind `UIRefreshControl` and positions it properly?

Reply:
Yeah, this was how codepath showed us how to implement refresh. https://guides.codepath.com/ios/Using-UIRefreshControl. Seems like the scroll view and table view operates similarly in this regard.